docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -156,7 +156,11 @@ rule's check interval.
 
 Alternatively, you can set the action frequency such that actions run for each
 alert. Choose how often the action runs (at each check interval, only when the
-alert status changes, or at a custom action interval). For {anomaly-detect}
+alert status changes, or at a custom action interval). 
+
+When you use an email action with an {anomaly-detect} rule, the alert includes only the top three anomalous records or influencers detected during the check interval. This behavior ensures that the alert remains concise by highlighting the most significant anomalies based on their scores.
```

Comment:
It is not specifically about email actions but rather about the overall alert context sent to any connector, as well as the document stored in the alert-as-data index.

I believe we should simply mention that `topRecords` and `topInfluencers` are limited to 3 documents.